source/rust_verify/src/main.rs

```diff
@@ -127,6 +127,33 @@ pub fn main() {
 
     let pervasive_path = our_args.pervasive_path.clone();
 
+    if our_args.error_report {
+        let mut args: Vec<String> = std::env::args().collect();
+        args.remove(0);
+
+        let index = args.iter().position(|x| *x == "--record").unwrap();
+        args.remove(index);
+
+        if let Some(verusroot) = &verus_root {
+            let exe = verusroot.path.join(if cfg!(windows) {
+                "error_report.exe"
+            } else {
+                "error_report"
+            });
+            if exe.exists() {
+                let mut res = std::process::Command::new(exe)
+                    .arg(&verusroot.path)
+                    .args(args)
+                    .spawn()
+                    .expect("running error_report");
+
+                res.wait().expect("error_report failed to run");
+                return;
+            }
+        }
```

Comment:
Don't fail silently, either panic (which I think is okay here, as you can expect a non-broken verus build) or print a message that says what went wrong.